alternatives.md

```diff
@@ -0,0 +1,129 @@
+# Babelfish vs Other Software
+
+Table of Contents
+=================
+
+* [Kythe](#kythe)
+* [Language Server Protocol](#language-server-protocol)
+* [srclib](#srclib)
+* [ctags](#ctags)
+* [ANTLR](#antlr)
+* [Tree\-sitter](#tree-sitter)
+* [srcML](#srcml)
+* [SmaCC](#smacc)
+
+<!-- TODO: https://github.com/oracle/opengrok/wiki/Comparison-with-Similar-Tools -->
+
+## [Kythe](http://kythe.io/)
+
+> The best way to view Kythe is as a “hub” for connecting tools for various languages, clients and build systems. By defining language-agnostic protocols and data formats for representing, accessing and querying source code information as data, Kythe allows language analysis and indexing to be run as services.
```

Comment:
Is this quote directly from the documentation?
Could you say it in the doc and add a link?